.env

```diff
@@ -43,8 +43,12 @@ ASSIGNMENT_STATE_INTERVAL_THRESHOLD=P1D
 LTI_LAUNCH_PRESENTATION_LOCALE=en-EN
 LTI_LAUNCH_PRESENTATION_RETURN_URL=http://localhost:8000/index.html
 LTI_OUTCOME_XML_NAMESPACE=http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0
+# Possible values for LTI Version: LTI-1p0 or 1.3.0
+LTI_VERSION=LTI-1p0
```

Comment:
There is naming inconsistency. Please use either `LTI-1p0` / `LTI-1p3` or `1.0.0` / `1.3.0`

Reply:
I would prefer it as well and in fact started with LTI-1p0 / LTI-1p3.
The problem here is that these are the names defined by IMS standards. If I change it here, it would probably change the value on the responses as well, unless I use different constants for this setting and for the responses. 
We need some agreement here. What do you think @Skyld @andreluizmachado

Reply:
My preference would be to use the format `1.x.x`. If the IMS spec is not consistent about the version formats, let's ignore it wherever we can (in configurations / API responses / code).

Reply:
Changed to 1.1.1 and 1.3.0